dlp/snippets/src/test/java/dlp/snippets/DeIdentificationTests.java

```diff
@@ -549,4 +549,34 @@ public void testDeIdentifyWithDeterministicEncryption() throws IOException {
     String output = bout.toString();
     assertThat(output).contains("Text after de-identification:");
   }
+
+  @Test
+  public void testDeIdentifyWithTimeExtraction() throws IOException {
```

Comment:
Please emulate: testDeIdentifyTableConditionMasking()
Bothe the input table and output are driven by the test case.
This makes the code less brittle.